src/index.js

```diff
@@ -177,6 +178,7 @@ module.exports = (
           {
             loader: eval('__dirname + "/loaders/ts-loader.js"'),
             options: {
+              transpileOnly,
```

Comment:
Looks good, we just need a test for this new behavior

Reply:
@styfle sure thing. Done.